homeassistant/components/dlna_dmr/media_player.py

```diff
@@ -345,6 +357,11 @@ async def async_config_update_listener(
         # Device was de/re-connected, state might have changed
         self.async_write_ha_state()
 
+    def async_write_ha_state(self) -> None:
```

Comment:
I'm not familiar with this method. Does `_attr_supported_features` need to be updated here as well as in `async_update`, or will one suffice?

Reply:
async_update calls `_async_write_ha_state`, and I didn't want to subclass that since its protected so we need it in both places